### ISSUE TITLE
Add fallback text to copy buttons

### ISSUE THREAD

**Issue by xavierdutreilh** [pypa/warehouse#861]
On the package detail page (e.g., https://warehouse.python.org/project/simplejson/), there is no text associated with the copy icon (next to the pip instructions):

```html
<a href="clicking-here-copies-text">
  <i class="fa fa-copy"></i>
</a>
```

So, screen readers completely fails to process it because the character \f0c5 injected by Font Awesome is unreadable.

A quick (and very much valid) fix would be to add a text alongside the icon:

```html
<a href="clicking-here-copies-text">
  <i class="fa fa-copy"></i>
  <span class="sr-only">Copy PIP instructions/<span>
</a>
```

Also, as mentionned in #810, it would be a good idea to add the attribute `aria-hidden="true"` to the `i` element.

**Comment by nlhkabu** [comment]
The same issue exists in the downloads table.

![screenshot from 2015-12-15 16-39-22](https://cloud.githubusercontent.com/assets/3323703/11823500/32e8cb36-a36c-11e5-9bf6-adee4d1686ea.png)

Here we should add:

1. `<span class="sr-only">MD5 Checksum Help</span>` for the help link
2. `<span class="sr-only">Copy MD5 Checksum</span>` for the MD5 link

**Comment by nlhkabu** [comment]
I have a fix for this on a local branch
